coding/using_eigen.rst

```diff
@@ -0,0 +1,11 @@
+###########
+Using Eigen
+###########
+
+.. _cpp_using_eigen:
+
+Eigen offers some **unsupported** modules which are located in the ``include/unsupported`` directory. For details, refer to the `relevant Eigen documentation`_.
```

Comment:
I'd like an opening sentence stating that people can use any part of eigen that is labeled supported before we talk about what is forbidden. This guide needs to tell people what they can do before telling them what they can't.